tests/OSPSuite.Presentation.Tests/Importer/Presenters/ColumnMappingPresenterSpecs.cs

```diff
@@ -55,14 +55,44 @@ public void identify_basic_format()
             () => _view.SetMappingSource(
                A<IList<ColumnMappingViewModel>>.That.Matches(l => 
                   l.Count() == 4 &&
-                  l.ElementAt(0).Equals(new ColumnMappingViewModel("Time", "Mapping,Time,min", new MappingDataFormatParameter("Time", new Column() { Name = Column.ColumnNames.Time, Unit = "min" }))) &&
-                  l.ElementAt(1).Equals(new ColumnMappingViewModel("Observation", "Mapping,Concentration,mol/l", new MappingDataFormatParameter("Observation", new Column() { Name = Column.ColumnNames.Concentration, Unit = "mol/l" }))) &&
-                  l.ElementAt(2).Equals(new ColumnMappingViewModel("Sp", "MetaData,Species", new MetaDataFormatParameter("Sp", "Species"))) &&
-                  l.ElementAt(3).Equals(new ColumnMappingViewModel("Study id", "GroupBy", new GroupByDataFormatParameter("Study id"))))
+
+                  l.ElementAt(0).IsEquivalentTo(new ColumnMappingViewModel("Time", "Mapping,Time,min", new MappingDataFormatParameter("Time", new Column() { Name = Column.ColumnNames.Time, Unit = "min" }))) &&
+                  l.ElementAt(1).IsEquivalentTo(new ColumnMappingViewModel("Observation", "Mapping,Concentration,mol/l", new MappingDataFormatParameter("Observation", new Column() { Name = Column.ColumnNames.Concentration, Unit = "mol/l" }))) &&
+                  l.ElementAt(2).IsEquivalentTo(new ColumnMappingViewModel("Sp", "MetaData,Species", new MetaDataFormatParameter("Sp", "Species"))) &&
+                  l.ElementAt(3).IsEquivalentTo(new ColumnMappingViewModel("Study id", "GroupBy", new GroupByDataFormatParameter("Study id"))))
             )).MustHaveHappened();
       }
    }
 
+   static public class ColumnMappingViewModelExtensions
+   {
+      static public bool IsEquivalentTo(this ColumnMappingViewModel self, ColumnMappingViewModel other)
+      {
+         if (self == null)
+            return other == null;
+         if (self.ColumnName != other.ColumnName || self.Description != other.Description)
+            return false;
+         switch (self.Source)
+         {
+            case MappingDataFormatParameter mp:
+               if (!(other.Source is MappingDataFormatParameter))
+                  return false;
+               var mmp = other.Source as MappingDataFormatParameter;
```

Comment:
It feels that this method should be implemented in EACH Parameter class instead of delegating that outside?

Reply:
> It feels that this method should be implemented in EACH Parameter class instead of delegating that outside?

Yes, you are right, but it is only needed for testing purposes so I would not put it on the production code and since it is only intended for one test, should we spend much more time on it? Maybe just a function instead of an extension method? I am open to suggestions here... I will do it as you recommend...

Reply:
oh I did not see that it was in the test project. sorry